packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1141,6 +1141,11 @@ The first three parameters in the table below correspond to different types of l
 
 Array of [User](#user-object)
 
+##### Known Limitation
+
+Due to an infrastructure limitation, [group administrators](https://help.okta.com/en-us/Content/Topics/Security/administrators-group-admin.htm), [help desk administrators](https://help.okta.com/en-us/Content/Topics/Security/administrators-help-desk-admin.htm),
```

Comment:
Can you describe the limitation (in the pull request, not the documentation)? Wondering if "infrastructure" is the right word choice.